reformCage.py

```diff
@@ -0,0 +1,177 @@
+"""
+From a cage .mol file, returns:
+
+	1) The SMILES code of the building blocks.
+	1.1) The active coordination number of the linkers.
+        1.2) The active functional group.
+        1.3) The number of times a building block has been used in a cage.
+
+	2) The topology of the cage.
+
+        3) Optional Arguments.
+        3.1) Optional (-c): Reform the unrelaxed cage from .mol file.
+        3.2) Optional (-b): Reform the unrelaxed building-blocks (bb).
+
+Usage: python reformCage.py <file.mol>
+
+Author: James T. Pegg
+
+"""
+import argparse
+import stk
+import rdkit
+from rdkit.Chem import AllChem
+import operator
+
+parser = argparse.ArgumentParser(description='Deconstruct cage.mol file')
+parser.add_argument('mol_file')
+parser.add_argument('-c', '--reform_cage',
+                    action='store_true',
+                    help=('Boolean: writes the unrelaxed cage to a .mol file.'))
```

Comment:
Don't specify type here. The user does not actually provide a typed variable when they give "-c" as an argument.